github/repos_commits.go

```diff
@@ -146,6 +147,35 @@ func (s *RepositoriesService) GetCommit(owner, repo, sha string) (*RepositoryCom
 	return commit, resp, err
 }
 
+// GetCommitSHA1 gets the SHA-1 of a commit reference.
+// If a last-known SHA1 is supplied and no new commits have occurred, a
+// 304 Unmodified response is returned.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/commits/#get-the-sha-1-of-a-commit-reference
+// See also: https://developer.github.com/changes/2016-02-24-commit-reference-sha-api/
```

Comment:
no need for this link since the GitHub docs already link to the announcement blog post.